BD-3790 Apply template standards to Custom Data (#8667)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/custom_data/event_naming_conventions.md

# Event naming conventions

> > Maintaining consistency in your event and attribute taxonomy during your Braze integration will keep your data clean and usable for new and existing users of the Braze platform. This helps to avoid issues later on that can trigger a campaign to the wrong audience or cause discrepancies in results from using the wrong event.<br><br>This page covers proper event naming conventions and best practices.

Reviewer: ```suggestion
> This page covers proper event naming conventions and best practices. By maintaining consistency in your event and attribute taxonomy, you'll keep your data clean and usable for new and existing users of the Braze platform. This helps avoid issues later, such as triggering a campaign to the wrong audience or generating wrong results after using the wrong event.
```
